standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnStore.java

```diff
@@ -131,6 +134,21 @@ enum MUTEX_KEY {
 
   int TIMED_OUT_TXN_ABORT_BATCH_SIZE = 50000;
 
+  String POOL_TX = "txnhandler";
+  String POOL_MUTEX = "mutex";
+  String POOL_COMPACTOR = "compactor";
```

Comment:
Can these be `static final` ?

Reply:
TxnStore is an interface so these are implicitly `public static final`.